### PR TITLE
Provide precompiled bundle

### PR DESCRIPTION
**Pull request by Nachtalb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "ftw.tagging",
+  "version": "1.0.0",
+  "main": "index.js",
```
Comment (tinagerber) [inline package.json]: ✂️ main

File: ftw/tagging/browser/configure.zcml [hunk 2]
```diff
@@ -5,7 +5,6 @@
     <browser:resourceDirectory
         name="ftw.tagging.stylesheets"
         directory="stylesheets"
-        layer=".interfaces.IFtwTaggingLayer"
```
Comment (tinagerber) [inline ftw/tagging/browser/configure.zcml]: Why did you remove this?

Comment (maethu) [reply ftw/tagging/browser/configure.zcml]: resourceDirectory does not support a layer.
